symbulate/plot.py

```diff
@@ -40,4 +41,117 @@ def plot(*args, **kwargs):
     except:
         plt.plot(*args, **kwargs)
         
+def is_discrete(heights):
+    return sum([(i > 1) for i in heights]) > .8 * len(heights)
+
+def count_var(x):
+    counts = {}
+    for val in x:
+        if val in counts:
+            counts[val] += 1
+        else:
+            counts[val] = 1
+    return counts
+    
+def compute_density(values):
+	density = gaussian_kde(values)
+	density.covariance_factor = lambda: 0.25
+	density._compute_covariance()
+	return density
+    
+def add_colorbar(fig, type, mappable):
+    if 'marginal' not in type:
+        caxes = fig.add_axes([0, 0.1, 0.05, 0.8])
+    else:
+        caxes = fig.add_axes([0, 0.1, 0.05, 0.57])
+        
+    cbar = plt.colorbar(mappable=mappable, cax=caxes)
+    caxes.yaxis.set_ticks_position('left')
+    cbar.set_label('Density')
+    caxes.yaxis.set_label_position('left')
+    return cbar, caxes
+
+def setup_tile(v, bins, discrete):
+    if not discrete:
+        v_bin = np.linspace(min(v), max(v) + 1, bins)
+        v_lab = [min(v)]
+        for i in range(len(v_bin) - 1):
+            v_lab.append(0.5 * (v_bin[i] + v_bin[i+1]))
+        v_pos = range(len(v_lab))
```

Comment:
Suggestion: Change `v_pos` to be [0.5, 1.5, 2.5, ...] instead of [0, 1, 2, ...] and change `v_lab` to be the actual endpoints of the bins (instead of the average.)

Reply:
Changing v_lab to the actual ends points makes it the same array as v_bin, doesn't it?

Reply:
Yes, I believe that's right.